src/routes/open/components/Layout.jsx

```diff
@@ -2,30 +2,73 @@
 import * as React from 'react'
 import ChevronLeft from '@material-ui/icons/ChevronLeft'
 import IconButton from '@material-ui/core/IconButton'
+import { withRouter } from 'react-router-dom'
+import queryString from 'query-string'
 import Stepper, { StepperPage } from '~/components/Stepper'
 import Block from '~/components/layout/Block'
 import Heading from '~/components/layout/Heading'
 import Row from '~/components/layout/Row'
 import Review from '~/routes/open/components/ReviewInformation'
 import SafeNameField from '~/routes/open/components/SafeNameForm'
 import SafeOwnersFields from '~/routes/open/components/SafeOwnersConfirmationsForm'
-import { getOwnerNameBy, getOwnerAddressBy, FIELD_CONFIRMATIONS } from '~/routes/open/components/fields'
+import {
+  getOwnerNameBy,
+  getOwnerAddressBy,
+  FIELD_CONFIRMATIONS,
+  FIELD_SAFE_NAME,
+} from '~/routes/open/components/fields'
 import { history } from '~/store'
 import { secondary } from '~/theme/variables'
 
 const getSteps = () => ['Name', 'Owners and confirmations', 'Review']
 
-const initialValuesFrom = (userAccount: string) => ({
-  [getOwnerNameBy(0)]: 'My Wallet',
-  [getOwnerAddressBy(0)]: userAccount,
-  [FIELD_CONFIRMATIONS]: '1',
-})
+const validateQueryParams = (ownerAddresses?: string[], ownerNames?: string[], threshold?: string, safeName?: string) => {
+  if (!ownerAddresses || !ownerNames || !threshold || !safeName) {
+    return false
+  }
+  if (!ownerAddresses.length === 0 || ownerNames.length === 0) {
+    return false
+  }
+  // eslint-disable-next-line no-restricted-globals
+  if (isNaN(threshold)) {
```

Comment:
https://github.com/airbnb/javascript#standard-library--isnan instead of disabling eslint rule

Reply:
done